fix(docs) Resolve 196 mdx lint warnings for just, quickly, actually, PostgreSQL (#47358)
supabase/supabase: apps/docs/content/troubleshooting/how-postgres-chooses-which-index-to-use-_JHrf4.mdx

An operator's functional equivalents, such as `IN`, `BETWEEN`, and `ANY`, are also valid.

> However, because the base requirements (relevant column, filter, and operators) are present, doesn't mean that an index will be used.

Reviewer: ```suggestion
However, an index isn't guaranteed to be used, even when the base requirements (relevant column, filter, and operators) are present.
```

The original change corrupted the meaning of "just because a, doesn't mean b" but it feels like you can actually reorder this sentence and put the important bit about an index not being used first.
